src/agent/helpers.ts

```diff
@@ -1,39 +1,13 @@
 import { ConnectionRecord } from '../modules/connections'
 import { AgentMessage } from './AgentMessage'
 import { OutboundMessage } from '../types'
-import { ConnectionInvitationMessage } from '../modules/connections'
-import { AriesFrameworkError } from '../error'
 
 export function createOutboundMessage<T extends AgentMessage = AgentMessage>(
```

Comment:
This method seems pretty useless now. It takes in two arguments and returns it as an object.

I'm generally not a fan of this method. Rather have OutboundMessage be a class instance in that case

Reply:
Yes, we can eventually get rid of this function. Perhaps by having something like `OutboundMessageContext`.